<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainW</class>
 <widget class="QWidget" name="MainW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>837</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>cluster cut control</string>
  </property>
  <widget class="QPushButton" name="openFileButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>84</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>load file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveFileButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>84</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>save file</string>
   </property>
  </widget>
  <widget class="QListWidget" name="hParamSelect">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>121</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="hChannelSelect">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>61</width>
     <height>141</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;select the channel for the horizontal axis to be displayed&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="vChannelSelect">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>61</width>
     <height>141</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;select the channel for the vertical axis to be displayed&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QListWidget" name="vParamSelect">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>121</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="workClusterSelect">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>61</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="viewClustersSelect">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>480</y>
     <width>61</width>
     <height>141</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
  </widget>
  <widget class="QLabel" name="chLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>channel</string>
   </property>
  </widget>
  <widget class="QLabel" name="hLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>H</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="vLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>V</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="paramLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>parameter</string>
   </property>
  </widget>
  <widget class="QLabel" name="viewingClusterLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>440</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;viewing clusters&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="workingClusterLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;working cluster&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="viewButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>710</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;update the data viewed in the plot window&lt;/p&gt;&lt;p&gt;keyboard shortcut: V&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>View</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>810</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>quit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>680</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;add a cluster with the points in the boundary&lt;/p&gt;&lt;p&gt;Keyboard shortcut: A&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refineButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>650</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;refine the selected working cluster with points in the boundaries&lt;/p&gt;&lt;p&gt;keyboard shortcut: R&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Refine</string>
   </property>
  </widget>
  <widget class="QPushButton" name="copyButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>710</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;add a cluster with the points in the boundary, but also keep the points in the original cluster also&lt;/p&gt;&lt;p&gt;keyboard shortcut: C&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>copy</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>openFileButton</sender>
   <signal>clicked()</signal>
   <receiver>MainW</receiver>
   <slot>loadFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewButton</sender>
   <signal>clicked()</signal>
   <receiver>MainW</receiver>
   <slot>updatePlotView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>724</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>809</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainW</receiver>
   <slot>quit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>16</x>
     <y>824</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>773</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>MainW</receiver>
   <slot>addCluster()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>698</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>748</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refineButton</sender>
   <signal>clicked()</signal>
   <receiver>MainW</receiver>
   <slot>refineCluster()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>662</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>638</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>workClusterSelect</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainW</receiver>
   <slot>changeWorkCluster()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>544</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>555</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hChannelSelect</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainW</receiver>
   <slot>invalidateView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>398</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vChannelSelect</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainW</receiver>
   <slot>invalidateView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>398</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hParamSelect</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainW</receiver>
   <slot>invalidateView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vParamSelect</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainW</receiver>
   <slot>invalidateView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>431</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewClustersSelect</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainW</receiver>
   <slot>invalidateView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>501</y>
    </hint>
    <hint type="destinationlabel">
     <x>128</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadFile()</slot>
  <slot>updatePlotView()</slot>
  <slot>quit()</slot>
  <slot>addCluster()</slot>
  <slot>refineCluster()</slot>
  <slot>changeWorkCluster()</slot>
  <slot>invalidateView()</slot>
 </slots>
</ui>
